src/doc_scan_service/session/create/check/requested.liveness.check.builder.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const RequestedLivenessConfig = require('./requested.liveness.config');
+const RequestedLivenessCheck = require('./requested.liveness.check');
+const DocScanConstants = require('../../../doc.scan.constants');
+const Validation = require('../../../../yoti_common/validation');
+
+/**
+ * Builder to assist the creation of {@link RequestedLivenessCheck}.
+ */
+class RequestedLivenessCheckBuilder {
+  constructor() {
+    this.maxRetries = 1;
+  }
+
+  /**
+   * Sets the type to be of a ZOOM liveness check
+   *
+   * @returns {this}
+   */
+  forZoomLiveness() {
+    return this.forLivenessType(DocScanConstants.ZOOM);
+  }
+
+  /**
+   * Sets the type of the liveness check to the supplied value
+   *
+   * @param {string} livenessType
+   *   The type of the liveness check
+   *
+   * @returns {this}
+   */
+  forLivenessType(livenessType) {
```

Comment:
It feels kinda weird for these to be for?

Is there a good reason they're not with? We use with to set strings in other places.

Reply:
It's just following the Java implementation - thoughts @echarrod / @MrBurtyyy ?

Reply:
I think `for` makes sense, as the config the check is _for_ a zoom liveness check, _with_ a maximum number of retries right?